src/lib/components/side-nav.svelte

```diff
@@ -64,6 +92,14 @@
         <div class="nav-title">{translate('archive')}</div>
       </NavRow>
     </IsCloudGuard>
```

Comment:
I'd put this NavRow in <IsCloudGuard> until we are ready for it in cloud. Or add a slot="batch"